packages/web/src/pages/chat-page/components/ReactionPopupMenu.tsx

```diff
@@ -0,0 +1,53 @@
+import { ComponentType, MutableRefObject } from 'react'
+
+import { reactionOrder, ReactionTypes } from '@audius/common'
+import { Popup, PopupPosition } from '@audius/stems'
+
+import {
+  reactionMap,
+  ReactionProps
+} from 'components/notification/Notification/components/Reaction'
+
+import styles from './ReactionPopupMenu.module.css'
+
+const reactionList: [ReactionTypes, ComponentType<ReactionProps>][] =
+  reactionOrder.map((r) => [r, reactionMap[r]])
+
+export const ReactionPopupMenu = ({
+  containerRef,
+  anchorRef,
+  isVisible,
+  onSelected,
+  onClose,
+  position = PopupPosition.BOTTOM_LEFT
+}: {
+  containerRef?: MutableRefObject<HTMLDivElement | null>
+  anchorRef: MutableRefObject<HTMLElement | null>
+  isVisible: boolean
+  onSelected?: (reaction: ReactionTypes) => void
+  onClose: () => void
+  position?: PopupPosition
+}) => {
+  return (
+    <Popup
+      containerRef={containerRef as MutableRefObject<HTMLDivElement>}
+      anchorRef={anchorRef as MutableRefObject<HTMLElement>}
+      isVisible={isVisible}
+      onClose={onClose}
+      className={styles.popup}
+      position={position}
+    >
+      <div className={styles.root}>
+        {reactionList.map(([reactionType, Reaction]) => (
+          <Reaction
+            key={reactionType}
+            onClick={(e) => {
+              onSelected?.(reactionType)
+            }}
+            isResponsive
```

Comment:
tbh not sure this needs to be marked `isResponsive`? cc: @dylanjeffers what does that do?

Reply:
it means it will respond to hover and press events, scaling essentially. probably this should be refactored to only have "responsive" if there in as onClick event? so in this case you do need it, but if you want to remove and refactor, that sounds great too!

Reply:
ah awesome, thanks! I was thinking it was responsive as in changes based on screen size 😅